.cspell/misc-terms.txt

```diff
@@ -1 +1,7 @@
 newbranch
+egdes
+BRANDES
+KOEPF
+affedcting
+handdrawn
+handdrawn
```

Comment:
I think:

- `egdes` is probably a typo of `edges`, so should be fixed in the code where it occurs.
- `affedcting` is probably a typo of `affecting`.
- `handdrawn` is probably a typo of `hand-drawn` or `hand drawn`.

Reply:
Will change, I was trying to get errors to 0 without any actual changes. 😆

Reply:
We'll have to keep handdrawn, as it's used in `handdrawnSeed: 0,`